node/websocket/websocket.go

```diff
@@ -480,14 +480,18 @@ func (c *connection) BlockByNumber(ctx context.Context, number uint64, full bool
 		return nil, errors.Wrap(err, "could not make request")
 	}
 
-	return c.parseBlockResponse(response)
+	return c.parseBlockResponse(response, n)
 }
 
-func (c *connection) parseBlockResponse(response *jsonrpc.RawResponse) (*eth.Block, error) {
+func (c *connection) parseBlockResponse(response *jsonrpc.RawResponse, lookup interface{}) (*eth.Block, error) {
 	if response.Error != nil {
 		return nil, errors.New(string(*response.Error))
 	}
 
+	if bytes.Equal(response.Result, json.RawMessage("null")) {
```

Comment:
Is it possible for Result to be "" or not provided in this scenario? (Worth checking for `len(response.Result) == 0`?)

Reply:
Added in beda050.